Sources/FoundationEssentials/String/String+Path.swift

```diff
@@ -170,11 +194,11 @@ extension String {
             return false
         }
         if let lastDot = pathExtension.utf8.lastIndex(of: UInt8(ascii: ".")) {
-            let beforeDot = pathExtension[..<lastDot].unicodeScalars
-            let afterDot = pathExtension[pathExtension.index(after: lastDot)...].unicodeScalars
+            let beforeDot = pathExtension[..<lastDot]._standardizingSlashes().unicodeScalars
+            let afterDot = pathExtension[pathExtension.index(after: lastDot)...]._standardizingSlashes().unicodeScalars
```

Comment:
Where do we expect an arc separator _after_ the extension separator? That is, the extension should be on the final arc, and so there would be no slashes right?

Reply:
We don't expect it to occur, but this is to ensure that we validate that it doesn't. I can't come up with any scenarios where we would currently end up in this situation, but `invalidExtensionScalars` contains a `/` and so to ensure that POSIX and Windows behaviors are consistent, I added this standardization here so that if we do somehow end up with a `\` in the path extension that we mark it as invalid on Windows